Build/packages.config

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Remotion.BuildScript" version="2.0.0-alpha024" targetFramework="net45" developmentDependency="true" />
+  <package id="Remotion.BuildScript" version="2.0.0-alpha025" targetFramework="net45" developmentDependency="true" />
```

Comment:
Should also update the release process script. I may be that it's required, and it's definitely a good idea by now.